.github/workflows/e2e-nvidia-l4-x1.yml

```diff
@@ -150,21 +150,8 @@ jobs:
 
       - name: Install ilab
         run: |
-          export CUDA_HOME="/usr/local/cuda"
-          export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CUDA_HOME/lib64:$CUDA_HOME/extras/CUPTI/lib64"
-          export PATH="$PATH:$CUDA_HOME/bin"
-          python3.11 -m venv --upgrade-deps venv
-          . venv/bin/activate
-          nvidia-smi
-          python3.11 -m pip cache remove llama_cpp_python
-
-          pip_install="python3.11 -m pip install -v -c constraints-dev.txt"
-          CMAKE_ARGS="-DGGML_CUDA=on" $pip_install .
-
-          # https://github.com/instructlab/instructlab/issues/1821
-          # install with Torch and build dependencies installed
-          $pip_install packaging wheel setuptools-scm
-          $pip_install .[cuda] -r requirements-vllm-cuda.txt
+          alias python='python3.11'
+          ./scripts/install-ilab-with-cuda.sh
```

Comment:
are aliases seen inside scripts? I think you have to export a variable.

Reply:
That I'm not 100% sure. I was going to try the alias thing first. If not, then yes, I can do something like

```
PYTHON=python3.11  ./scripts/install-ilab-with-cuda.sh
```

and have the script use `PYTHON` to call the python executable

Reply:
yes variables are the way to go